src/components/OrderDetails/OrderDetails.module.css

```diff
@@ -0,0 +1,95 @@
+.details {
+  background-color: antiquewhite;
```

Comment:
I would recommend using ordering CSS. Such ordered styles are easier to read for everyone:

- you now (and you in the future),
- your colleagues,
- someone doing CR of your code,
- everyone who is interesting in reading it.

It's great especially when all people in the project do it the same way. An easy proposition for example: https://9elements.com/css-rule-order/